McAlisterWing/mcalister.i

```diff
@@ -239,18 +217,18 @@ realms:
 
     restart:
       restart_data_base_name: rst/mcalister.rst
-      restart_frequency: 1001
-      restart_start: 1001
+      restart_frequency: 50
+      restart_start: 5
```

Comment:
@jhux2 Would be good to change this to a number greater than the number of timesteps for timing studies to avoid I/O messing up the timing studies.